src/habit.ts

```diff
@@ -39,22 +41,28 @@ const locationToString = (locationPreposition: LocationPreposition): string => {
   }
 }
 
-export type Habit = {
-  id: string
+export type HabitSentence = {
   habitText: string
   timePreposition: TimePreposition
   timeText: string
   locationPreposition: LocationPreposition
   locationText: string
+}
+
+export type Habit = {
```

Comment:
Do we need to save all the parts for anything? it might makes sense to just save the stringified sentence:
```suggestion
export type Habit = {
  id: string
  habitSentence: string
  history: Posix[]
  }
```

Reply:
I was thinking it would be good to save each piece in case we want to repopulate this kind of screen to edit a habit. Thoughts?